Procfile

```diff
@@ -1 +1 @@
-web: RESOURCE_INDEX=$(RESOURCE_INDEX_PREFIX="$HEROKU_BRANCH" ./get-resource-index) node server.js --verbose
+web: RESOURCE_INDEX=$(RESOURCE_INDEX_PREFIX="$HEROKU_BRANCH" node ./scripts/get-resource-index.js) node server.js --verbose
```

Comment:
Do you have some docs on `HEROKU_BRANCH`? I can only find [this page](https://devcenter.heroku.com/articles/github-integration-review-apps) which implies it's only for review apps, but also implies that it's not stable and is likely to change in the future.

It'd be clearer to prepend the server call with a env-setting function, like `set-review-app-variables` which could self-document the steps that are going on, and perhaps query other env variables to ensure we are indeed running on a review app.

Reply:
Note that we already use one of those variables, `HEROKU_PR_NUMBER`, to detect review app-ness:

https://github.com/nextstrain/nextstrain.org/blob/31ecbc4a7b9d5ab72341040a1e06371551cec659/src/config.js#L36-L46

 and adjust config:

https://github.com/nextstrain/nextstrain.org/blob/31ecbc4a7b9d5ab72341040a1e06371551cec659/src/config.js#L391-L412

Reply:
I was hesitant to suggest that because it means that the value passed in as `RESOURCE_INDEX` won't be what's actually used depending on unseen other values (i.e. `HEROKU_PR_NUMBER`), and that seemed undesirable/confusing and a first for the config. I think `RESOURCE_INDEX` for review apps should still be set outside of the app, like it is for production.

Reply:
No longer relevant for this PR since I'm going to do the review-app stuff separately.